src/openrct2/object/RideObject.cpp

```diff
@@ -969,155 +970,156 @@ bool RideObject::IsRideTypeShopOrFacility(uint8_t rideType)
+                                                   { "lay_down_rc", RIDE_TYPE_LAY_DOWN_ROLLER_COASTER },
+                                                   { "suspended_monorail", RIDE_TYPE_SUSPENDED_MONORAIL },
+                                                   { "reverser_rc", RIDE_TYPE_REVERSER_ROLLER_COASTER },
+                                                   { "heartline_twister_rc", RIDE_TYPE_HEARTLINE_TWISTER_COASTER },
+                                                   { "mini_golf", RIDE_TYPE_MINI_GOLF },
+                                                   { "giga_rc", RIDE_TYPE_GIGA_COASTER },
+                                                   { "roto_drop", RIDE_TYPE_ROTO_DROP },
+                                                   { "flying_saucers", RIDE_TYPE_FLYING_SAUCERS },
+                                                   { "crooked_house", RIDE_TYPE_CROOKED_HOUSE },
+                                                   { "monorail_cycles", RIDE_TYPE_MONORAIL_CYCLES },
+                                                   { "compact_inverted_rc", RIDE_TYPE_COMPACT_INVERTED_COASTER },
+                                                   { "water_coaster", RIDE_TYPE_WATER_COASTER },
+                                                   { "air_powered_vertical_rc", RIDE_TYPE_AIR_POWERED_VERTICAL_COASTER },
+                                                   { "inverted_hairpin_rc", RIDE_TYPE_INVERTED_HAIRPIN_COASTER },
+                                                   { "magic_carpet", RIDE_TYPE_MAGIC_CARPET },
+                                                   { "submarine_ride", RIDE_TYPE_SUBMARINE_RIDE },
+                                                   { "river_rafts", RIDE_TYPE_RIVER_RAFTS },
+                                                   { "enterprise", RIDE_TYPE_ENTERPRISE },
+                                                   { "inverted_impulse_rc", RIDE_TYPE_INVERTED_IMPULSE_COASTER },
+                                                   { "mini_rc", RIDE_TYPE_MINI_ROLLER_COASTER },
+                                                   { "mine_ride", RIDE_TYPE_MINE_RIDE },
+                                                   { "lim_launched_rc", RIDE_TYPE_LIM_LAUNCHED_ROLLER_COASTER },
+                                                   { "hypercoaster", RIDE_TYPE_HYPERCOASTER },
+                                                   { "hyper_twister", RIDE_TYPE_HYPER_TWISTER },
+                                                   { "monster_trucks", RIDE_TYPE_MONSTER_TRUCKS },
+                                                   { "spinning_wild_mouse", RIDE_TYPE_SPINNING_WILD_MOUSE },
+                                                   { "classic_mini_rc", RIDE_TYPE_CLASSIC_MINI_ROLLER_COASTER },
+                                                   { "hybrid_rc", RIDE_TYPE_HYBRID_COASTER },
+                                                   { "single_rail_rc", RIDE_TYPE_SINGLE_RAIL_ROLLER_COASTER } };
```

Comment:
This is way too far intended. Is there no way to just have it indented one extra level?

Reply:
There is, just add a trailing comma, just like the other lists have.
```suggestion
                                                   { "single_rail_rc", RIDE_TYPE_SINGLE_RAIL_ROLLER_COASTER }, };
```